app/screens/settings/display_theme/theme_thumbnail.tsx

```diff
@@ -0,0 +1,296 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import Svg, {Rect, G, Circle} from 'react-native-svg';
+
+type ThemeThumbnailProps = {
+    awayIndicator: string;
+    borderColorBase: string;
+    borderColorMix: string;
+    buttonBg: string;
+    centerChannelBg: string;
+    centerChannelColor: string;
+    dndIndicator: string;
+    newMessageSeparator: string;
+    onlineIndicator: string;
+    sidebarBg: string;
+    sidebarText: string;
+    sidebarUnreadText: string;
```

Comment:
I just not get why don't we pass the `theme` as the prop

Reply:
corrected